helpers/presentation_objects.py

```diff
@@ -252,7 +252,7 @@ def format_label(name, units=None):
 
     Slight variance on _title() in iris/quickplot.py.
     """
-    label = name.replace("_", " ").capitalize()
+    label = name.replace("_", " ")
```

Comment:
see the above comment regarding `replace()`

Reply:
Yes, see above.